pkg/api/types.go

```diff
@@ -1481,17 +1489,13 @@ const (
 	// onto the node entirely. Enforced by the scheduler.
 	TaintEffectPreferNoSchedule TaintEffect = "PreferNoSchedule"
 	// NOT YET IMPLEMENTED. TODO: Uncomment field once it is implemented.
-	// Do not allow new pods to schedule onto the node unless they tolerate the taint,
-	// do not allow pods to start on Kubelet unless they tolerate the taint,
+	// Do not allow pods to start on Kubelet unless they tolerate the taint,
 	// but allow all already-running pods to continue running.
-	// Enforced by the scheduler and Kubelet.
-	// TaintEffectNoScheduleNoAdmit TaintEffect = "NoScheduleNoAdmit"
-	// NOT YET IMPLEMENTED. TODO: Uncomment field once it is implemented.
-	// Do not allow new pods to schedule onto the node unless they tolerate the taint,
-	// do not allow pods to start on Kubelet unless they tolerate the taint,
-	// and evict any already-running pods that do not tolerate the taint.
-	// Enforced by the scheduler and Kubelet.
-	// TaintEffectNoScheduleNoAdmitNoExecute = "NoScheduleNoAdmitNoExecute"
+	// Enforced by Kubelet.
+	// TaintEffectNoAdmit TaintEffect = "NoAdmit"
+	// Evict any already-running pods that do not tolerate the taint.
+	// Enforced by Kubelet.
```

Comment:
This is actually enforced by NodeController in this PR. But some taint evictions might be done by Kubelet in the future. So you should just say here "Currently enforced by NodeController."